### DISCUSSION TITLE
How can I get table alias name from DataReader?

### DISCUSSION THREAD

**Discussion by AlexKerman** [mysql-net/MySqlConnector#1450] [Q&A] [answered]
I have user-written query with unlimited joins. Some joins can refer same table.
I need to know which alias current column belongs to. 
MySqlDataReader have private field m_resultSet. ResultSet have ColumnDefinitions. ColumnDefinition have Table property what exactly I need.
How I can access it?

**Comment by bgrainger** [comment]
Does `MySqlDataReader.GetColumnSchema` return the information you're looking for? https://learn.microsoft.com/en-us/dotnet/api/system.data.common.dbdatareaderextensions.getcolumnschema?view=net-8.0

**Reply by AlexKerman** [reply]
Nope. MySqlDbColumn have only table physical name, not alias. And have no fields for it.
Table field (ColumnDefinitionPayload) have no references at all.

**Comment by AlexKerman** [comment, marked as answer]
Closed as far as #1521
